ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController_SwiftUI.swift

```diff
@@ -104,6 +116,18 @@ struct ButtonDemoView: View {
         })
     }
 
+    @ViewBuilder
+    private func demoToggle(_ controlSize: ControlSize, isDisabled: Bool) -> some View {
+        Toggle(isOn: $isToggleOn, label: {
+            buttonLabel
+        })
+        .toggleStyle(FluentButtonToggleStyle())
+        .controlSize(controlSize)
+        .disabled(isDisabled)
+        .fixedSize()
+        .padding(8.0)
```

Comment:
supernit: maybe this could be `GlobalTokens.spacing(.size80)`? Using our tokens library whenever applicable including in our demo files should encourage clients to do the same.